Changelog.md

```diff
@@ -4,6 +4,7 @@
 
 - In properties file, completion for possible enum values and booleans of a Camel component property 
 - In Properties file, the default values are automatically added when auto-completing Camel component properties
+- In Properties file, provide filtered completion when in middle of a component id, component property or value. Previously, completion was available only right after the dot or equal separators.

```

Comment:
btw. now that I read it again. Why do we have properties sometimes with capital P and sometimes with lower case p?

Reply:
because I wasn't coherent :-(
I will update it in this PR

Reply:
done